scripts/test-sandbox.sh

```
@@ -856,6 +856,300 @@ $CLI cloud sandbox snapshot delete "$SNAPSHOT_ID" --confirm 2>/dev/null || true
+BUILD_DIR="$TEST_DIR/build-test"
+mkdir -p "$BUILD_DIR/scripts"
+mkdir -p "$BUILD_DIR/config"
+echo "app.js content" > "$BUILD_DIR/app.js"
+echo "helper.js content" > "$BUILD_DIR/scripts/helper.js"
+echo '{"key": "value"}' > "$BUILD_DIR/config/settings.json"
+echo "should be excluded" > "$BUILD_DIR/exclude.png"
+pass "Build test files created"
+
+# Test: Basic build with files
+info "Test: snapshot build - basic with files"
+BUILD_NAME="build-test-$(date +%s)"
```

Shellcheck (0.11.0)

[warning] BUILD_NAME appears unused. Verify use (or export if used externally).

(SC2034)